docs/teams/mentoring.md

```diff
@@ -37,32 +38,15 @@ original:
 > There are currently five Local Team Coordinators who cover a rough geographic area of the country.  They will be responsible for you as a Mentor and will hopefully be able to arrange social events throughout the year where you can meet other Mentors.
 >
 > Think this is something you would like to be involved in?  We would ideally allocate a Mentor to each of the 54 teams so we need as many of you to volunteer as possible…so get involved and help guide a team to success in this year’s competition!
->
-> If you would like to be considered for this role we require the following information:
->
-> Name?
->
-> Location (where you live/work if this is different)?
->
-> How far are you willing to travel?
->
-> Roughly how often could you commit to visiting a school?
->
-> What skills do you possess which would make you a suitable mentor?
->
-> Have you had any previous experience with SR kits/teams e.g. as a volunteer or a participant?
->
-> Please confirm that you are over 18.
 
 
-* Once replies obtained, make ‘Mentor Sign-Up’ spreadsheet with this info and share with LTCs so that they can review and match Mentors with their teams.
+* Once replies obtained, make ‘Mentor Sign-Up’ spreadsheet with this info and match Mentors with their teams.
```

Comment:
```suggestion
Once we have the replies, make ‘Mentor Sign-Up’ spreadsheet with this info and match Mentors with their teams.

The steps are then:

```
Maybe also clarify whether that's replies from volunteers or from teams (or is it both?)